packages/@okta/vuepress-site/docs/guides/index.md

```diff
@@ -55,22 +55,68 @@ guides:
+
+    * [OAuth 2.0 and OpenID Connect overview](/docs/concepts/oauth-openid/)
+    * [Authorization servers](/docs/concepts/auth-servers/)
+    * [Policies](/docs/concepts/policies/)
+    * [Quickstart: Signing in your first user](/docs/guides/quickstart/cli/create-org/)
+    * [Set up self-service registration](/docs/guides/set-up-self-service-registration/before-you-begin/)
+
+2. Sign users in
+
+    * [Sign users in to your single-page application](/docs/guides/sign-into-spa/angular/before-you-begin/)
+    * [Sign users in to your web application](/docs/guides/sign-into-web-app/aspnet/before-you-begin/)
+    * [Sign users in using the Okta Sign-In Widget](/code/javascript/okta_sign-in_widget/)
+    * [Configure Okta sign-on and App sign-on policies](/docs/guides/configure-signon-policy/before-you-begin/)
+    * [Sign users out](/docs/guides/sign-users-out/android/before-you-begin/)
+
+3. Authorize
+
+    * [Implement the Authorization Code Flow](/docs/guides/implement-auth-code/overview/)
+    * [Implement the Authorization Code Flow with PKCE](/docs/guides/implement-auth-code-pkce/overview/)
+    * [Implement the Client Credentials Flow](/docs/guides/implement-client-creds/overview/)
+    * [Refresh access tokens](/docs/guides/refresh-tokens/overview/)
+    * [Validate access tokens](/docs/guides/validate-access-tokens/go/overview/)
+    * [Validate ID tokens](/docs/guides/validate-id-tokens/overview/)
+    * [Configure an access policy](/docs/guides/configure-access-policy/overview/)
+    * [Customize tokens returned from Okta with custom claims](/docs/guides/customize-tokens-returned-from-okta/overview/)
+    * [Customize tokens returned from Okta with a Groups claim](/docs/guides/customize-tokens-groups-claim/overview/)
+    * [Customize tokens returned from Okta with a dynamic whitelist](/docs/guides/customize-tokens-dynamic/overview/)
+    * [Customize tokens returned from Okta with a static whitelist](/docs/guides/customize-tokens-static/overview/)
+
+4. Brand and customize
+
+    * [Style the Widget](/docs/guides/style-the-widget/before-you-begin/)
+    * [Customize the Okta URL domain](/docs/guides/custom-url-domain/overview/)
+    * [Customize SMS messages](/docs/guides/sms-customization/before-you-begin/)
+    * [Customize email notifications and email domains](/docs/guides/email-customization/before-you-begin/)
+
+5. Deploy to production
+
+    * [Pre-launch checklist](/docs/guides/production-deployment/deployment-checklist/)
+    * [Deploy your app](/docs/guides/deploy-your-app/overview/)
+    * [Migrate to Okta](/docs/guides/migrate-to-okta/prerequisites/)
+
+## Publish an integration
 
-## Featured Guides
+Partner integrations connect your app or service to our mutual customers. One of the most common integrations is Single Sign-On (SSO): giving Okta users the ability to sign in directly to your application through Okta. Integrations can also extend Okta's functionality or integrate with your service in more complex ways.
```

Comment:
Many partners also build provisioning integrations (using the SCIM protocol) to automate lifecycle management use cases for their customers.